package.json

```diff
@@ -106,5 +106,8 @@
     "> 0.1%",
     "ie 8",
     "ie 9"
-  ]
+  ],
+  "dependencies": {
+    "@automattic/puppeteer-utils": "github:Automattic/puppeteer-utils#0f3ec50"
```

Comment:
I think this intent to be a `devDependencies`

Reply:
@claudiosanches I will double check but IIRC when I was working on this, the package dev dependencies are for development of the package in this repo while the dependencies are the packages it needs to run as a package in a down stream project.

Reply:
@claudiosanches Fixed up in https://github.com/woocommerce/woocommerce/pull/27113/commits/e78beaf5693609140dfcd7ed75ab76e6e631cb3a It was added to the wrong package.json.